packages/fuchsia_ctl/bin/main.dart

```diff
@@ -37,9 +37,9 @@ Future<void> main(List<String> args) async {
         abbr: 'd',
         help: 'The device node name to use. '
             'If not specified, the first discoverable device will be used.')
-    ..addOption('dev-finder-path',
-        defaultsTo: './dev_finder',
-        help: 'The path to the dev_finder executable.')
+    ..addOption('device-finder-path',
```

Comment:
Will this require updating the shell script on CI?

Reply:
Looks like https://flutter.googlesource.com/recipes/+/refs/heads/master/recipe_modules/fuchsia_util/api.py#74 will need updating, but the test shell script doesn't use this option so that's fine.

Reply:
I'll send the CL to update fuchsia_util/api.py. By the way we already have presubmit tasks for fuchsia_ctl but somehow the checks were not created.